optimum/habana/transformers/models/video_llava/processing_video_llava.py

```diff
@@ -0,0 +1,100 @@
+from typing import List, Optional, Union
+import numpy as np
+
+from transformers.models.video_llava.processing_video_llava import VideoLlavaProcessor
+from transformers.image_processing_utils import BatchFeature
+from transformers.image_utils import ImageInput, get_image_size, to_numpy_array
+from transformers.tokenization_utils_base import PaddingStrategy, PreTokenizedInput, TextInput, TruncationStrategy
+from transformers.utils import TensorType
+from optimum.utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+
+class GaudiVideoLlavaProcessor(VideoLlavaProcessor):
+    attributes = ["image_processor", "tokenizer"]
+    valid_kwargs = ["chat_template", "patch_size", "vision_feature_select_strategy", "image_token", "video_token"]
+    image_processor_class = "VideoLlavaImageProcessor"
+    tokenizer_class = "AutoTokenizer"
+
+    def __init__(
+        self,
+        image_processor=None,
+        tokenizer=None,
+        patch_size=None,
+        vision_feature_select_strategy=None,
```

Comment:
```suggestion
        patch_size=14,
        vision_feature_select_strategy="default",
```

@kaixuanliu is there a reason you changed the default values?

Reply:
Hi, I did not change the default value, I just used the code from transformers 4.45.2 [L61](https://github.com/huggingface/transformers/blob/v4.45.2/src/transformers/models/video_llava/processing_video_llava.py#L61), as the modeling forward part is based on transformers 4.45.2

Reply:
@kaixuanliu Can you check if Transformers v4.49 brought any change please? Since we are aligned on this version of Transformers at the moment.

Reply:
Sure. In transformers 4.49, VideoLlavaProcessor will expand `prompt_strings`, which will cause `inputs_not_expanded` in [L301](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/models/video_llava/modeling_video_llava.py#L301) to be false, hence  causes `legacy_processing` in [L324](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/models/video_llava/modeling_video_llava.py#L324) to be `False`, and will lead to call `masked_scatter` op in [L423](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/models/video_llava/modeling_video_llava.py#L423), this op in Gaudi3 will cost >2000ms. While in transformers 4.45.2, the input expanding operation is done in modeling part.

Reply:
So the 4.45 implementation is better in terms of perf on Gaudi3 ?

Reply:
Yes, it's 51 tokens/second compared with 22 tokens/second from 4.49